bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "yarn-sdk-behavior",
-  "version": "3.0.1",
+  "version": "3.1.2",
```

Comment:
I've been learning the hard way about correctly doing releases, I believe my local version of bower.json had 3.1.1 already and I just bumped that. It's now 3.1.3 since I bumped on the branch commit, then again after merging into master. I now know to wait until everything is in master before messing with it.